package.json

```diff
@@ -26,6 +26,7 @@
     "webpack-cli": "^4.9.1"
   },
```

Comment:
- [ ] The _dervServer_ package is missing your application. Kindly install the same and add that in your **webpack** configuration to enable your application to be run on the server locally.

**devServer in webpack configuration file:**
![image](https://user-images.githubusercontent.com/12745474/134922749-d0016f56-002e-4415-8882-4eb3dcbf6551.png)